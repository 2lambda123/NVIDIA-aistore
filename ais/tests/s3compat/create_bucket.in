aws --endpoint-url http://localhost:8080/s3 s3 mb s3://$BUCKET
aws --endpoint-url http://localhost:8080/s3 s3 ls
aws --endpoint-url http://localhost:8080/s3 s3 ls s3://$BUCKET
aws --endpoint-url http://localhost:8080/s3 s3 rb s3://$BUCKET
