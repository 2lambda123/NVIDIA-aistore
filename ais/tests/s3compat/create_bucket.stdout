make_bucket: $BUCKET
^(.*) $BUCKET$
remove_bucket: $BUCKET
